src/MeshlessFV/MfvMusclSimulation.cpp

```diff
@@ -207,18 +264,25 @@ void MfvMusclSimulation<ndim>::MainLoop(void)
   // Update all active cell counters in the tree
   mfvneib->UpdateActiveParticleCounters(partdata, mfv);
 
+  mfv->CopyDataToGhosts(simbox, partdata);
+#ifdef MPI_PARALLEL
+  MpiGhosts->CopyHydroDataToGhosts(simbox,mfv);
+#endif
+
 
   //Calculate all properties (and copy updated data to ghost particles)
   mfvneib->UpdateAllProperties(mfv->Nhydro, mfv->Ntot, partdata, mfv, nbody, simbox);
   mfv->CopyDataToGhosts(simbox, partdata);
+#ifdef MPI_PARALLEL
+  MpiGhosts->CopyHydroDataToGhosts(simbox,mfv);
+#endif
 
   // Calculate all matrices and gradients (and copy updated data to ghost particles)
   // TODO:
   //   Compute gradients for all cells neighbouring active ones (use levelneib?).
   mfvneib->UpdateGradientMatrices(mfv->Nhydro, mfv->Ntot, partdata, mfv, nbody, simbox);
   mfv->CopyDataToGhosts(simbox, partdata);
```

Comment:
This call must be redundant since we don't copy the data to mpi ghosts. After all we don't use the periodic ghosts in either the flux or the gravity routines.